src/components/presentational/DataSummary/DataSummary.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { CSSProperties } from 'react';
+import './DataSummary.css';
+import { UnstyledButton } from '..';
+import { ColorDefinition } from '../../../helpers/colors';
+
+export interface DataSummaryProps {
```

Comment:
The name of this seems a bit too broad; how about "DeviceDataPointSummary" or "DeviceDataPoint", or "SingleDataPoint" or something of that nature...since like a data summary could be anything

Reply:
I think I like `SingleDataPoint` out of those options because it can be used for non-device data, too, like AQI or computed values (or whatever else you want).

I'll get it renamed.

Reply:
Renamed to `SingleDataPoint`.